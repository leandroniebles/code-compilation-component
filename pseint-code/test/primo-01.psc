Proceso sin_titulo
	leer N
	Si N<0  entonces 
		Nu<-N*(-1)
	SiNo
		Nu<-N
	FinSi
	Si N mod 2 = 0 Entonces 
		Escribir 'Numero Primo:',Nu=2
		Si Nu<>2 Entonces
			Escribir N,'=2x',N/2
		FinSi
	Sino
		EsPrimo<-Nu<>1 
		Nu<-RC(Nu)
		f<-3
		Mientras f<=Nu Y EsPrimo Hacer
			Si N%F=0 Entonces 
				EsPrimo<-Falso
			FinSi
			f<-f+2
		FinMientras
		Escribir 'Numero Primo:',EsPrimo
		Si N>1 Y NO EsPrimo Entonces 
			Escribir N,'=',f-2,'x',N/(f-2)
		FinSi
	FinSi
FinProceso
