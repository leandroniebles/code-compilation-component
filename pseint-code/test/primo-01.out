> 123
Numero Primo:FALSO
123=3x41
